module/TaskManagement/config/module.config.php

```diff
@@ -39,6 +40,20 @@
 					),
 				),
 			),
+			'reminder-share-assignment' => array(

```

Comment:
"reminder-share-assignment" non è corretto rispetto alla semantica della mappatura. Essa non mappa solo il reminder dell'assegnazione degli shares, ma i reminders sui task in generale quindi "task-reminders" sarebbe più appropriato